js/functions/spotifyColors.js

```diff
@@ -0,0 +1,20 @@
+/* global app */
+
+window.functions.spotifyColors = {
+  name: 'spotify colors',
+  type: 'Edit',
+  run: async function () {
+    const token = window.localStorage.getItem('spotifyToken')
+    if (!token) {
+      const answer = window.confirm('Do you want to authenticate Spotify?')
+      console.log(answer)
+      if (answer) {
+        window.Sent.spotifyAuth('d41b83c2f2224356bd94fbf83924e6d4', 'https://artware.app')
```

Comment:
think this should be using the new `spotifyMoodColors` library right? not the old `Sent` library,
```js
window.SpotifyMoodColors.spotifyAuth('d41b83c2f2224356bd94fbf83924e6d4', 'https://artware.app')
```